articles/communication-services/concepts/service-limits.md

```diff
@@ -78,6 +78,7 @@ For more information on the SMS SDK and service, see the [SMS SDK overview](./sm
 |Number of participants in thread|250 |
 |Batch of participants - CreateThread|200 |
 |Batch of participants - AddParticipant|200 |
+|List of messages - ListMessages|200 |
```

Comment:
nit: "Page size - ListMessages" would be a bit more consistent with the other items

Reply:
Updated

Reply:
I don't have the permission to merge this change? Could you merge it for me? Thanks